icd-503-high.yaml

```diff
@@ -0,0 +1,404 @@
+# Based on the ICD-503 High-High-High Security Controls
+# Current as of 15-June-2018
+# Maintained by Mark Shoger (mshoger@redhat.com)
+# https://www.dni.gov/files/documents/ICD/ICD_503.pdf
+name: ICD-503 H-H-H
```

Comment:
Should the dash in the names be kept? Consistently seeing it referred to as ``ICD 503`` vs ``ICD-503``

ref: 
https://fas.org/irp/dni/icd/icd-503.pdf